### DISCUSSION TITLE
Secure Seed

### DISCUSSION THREAD

**Discussion by xKumorio** [PaperMC/Paper#10506] [Ideas]
### Is your feature request related to a problem?

So that players don't abuse the use of seed

### Describe the solution you'd like.

You can slightly modify the generation so that you can't use the seed of the world
or this https://github.com/Earthcomputer/SecureSeed

### Describe alternatives you've considered.

idk

### Other

-

**Comment by electronicboy** [comment]
I believe that we generally decided against this because it's a fairly set of invasive changes to solve something which is relatively low priority vs all of other concerns for an issue we already have some level of mitigation for

**Comment by 456dev** [comment]
If changing the structure / generation seeds in the config files isn't enough, you can always use a datapack/plugin to change worldgen
